src/lib/components/inbox/conversations/conversation_snippet.tsx

```diff
@@ -109,7 +111,7 @@ export class ConversationSnippet extends React.Component<Props, any> {
     const artworkDate = `${artwork.date}`
     const artworkArtist = `${artwork.artist.name} · `
     const conversationText = conversation.last_message.replace(/\n/g, " ")
-    const date = "11:00 AM"
+    const date = moment(conversation.last_message_at).fromNow(true)
```

Comment:
yeah I saw that but, and please correct me if I'm wrong, if it's 1pm now, and last message was 2 hours ago, 11am, will the string be `11:00 AM` or `2 hours ago`?